src/Suave.Tests/Cookie.fs

```diff
@@ -113,6 +113,7 @@ let setCookie (_ : SuaveConfig) =
           secure    = true
           httpOnly  = false }
       let ctx = Cookie.setCookie cookie { HttpContext.empty with runtime = { HttpRuntime.empty with logger = log }}
+      System.Threading.Thread.Sleep 1000
```

Comment:
We should use one of the info**WithAck** functions instead, if want to assert on the log message.